.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
Cargo.lock file should not be in the repo and commited.  Please use the gitignore from this https://www.toptal.com/developers/gitignore/api/rust